packages/frontend/src/components/Charts/CrabStrategyV2PnLChart.tsx

```diff
@@ -2,18 +2,24 @@ import { Box, CircularProgress, Typography} from '@material-ui/core'
 import { createStyles, makeStyles } from '@material-ui/core/styles'
 import React, { memo, useEffect } from 'react'
 import useAppMemo from '@hooks/useAppMemo'
-import { useCrabPnLV2ChartData } from 'src/state/ethPriceCharts/atoms'
+import { crabv2StrategyFilterEndDateAtom, crabv2StrategyFilterStartDateAtom, useCrabPnLV2ChartData } from 'src/state/crab/atoms'
 import { crabV2graphOptions } from '@constants/diagram'
 import HighchartsReact from 'highcharts-react-official'
 import Highcharts from 'highcharts'
-
+import Grid from "@material-ui/core/Grid";
+import MomentUtils from "@date-io/moment";
```

Comment:
Can we use date-fns instead? Will it create any issue?

Reply:
Changed